static/app/views/replays/utils/mergeReplayEntries.tsx

```diff
@@ -0,0 +1,11 @@
+import {Replay} from '../types';
+
+export default function mergeReplayEntries(array: Replay[], key: string) {
+  const initialValue = {};
+  return array.reduce((obj, item) => {
+    return {
+      ...obj,
+      [item[key]]: item,
+    };
+  }, initialValue);
```

Comment:
Although `array` here will be small, using reduce with object spread is a slow method to create an object. I think a better (in perf and readability imo) way is:

```suggestion
  return Object.fromEntries(array.map((obj, item) => [item[key], item]));
```

Reply:
And imo since it's used once and is a simple one liner now, we probably don't need a new function for this, it can be done in replayTable